packages/snap/package.json

```diff
@@ -23,21 +23,24 @@
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' --ignore-path .gitignore",
     "serve": "mm-snap serve",
-    "start": "mm-snap watch"
+    "start": "mm-snap watch",
+    "test": "jest"
   },
   "dependencies": {
     "@metamask/snaps-types": "^0.32.2",
     "@metamask/snaps-ui": "^0.32.2",
     "buffer": "^6.0.3"
   },
   "devDependencies": {
+    "@jest/globals": "^29.5.0",
     "@lavamoat/allow-scripts": "^2.0.3",
     "@metamask/auto-changelog": "^2.6.0",
     "@metamask/eslint-config": "^10.0.0",
     "@metamask/eslint-config-jest": "^10.0.0",
     "@metamask/eslint-config-nodejs": "^10.0.0",
     "@metamask/eslint-config-typescript": "^10.0.0",
     "@metamask/snaps-cli": "^0.32.2",
+    "@metamask/snaps-jest": "^0.35.2-flask.1",
```

Comment:
Should we be using the Flask versions for the CLI too? 🤔

Reply:
I don't think there's any significant difference between `0.32.2` and `0.35.2-flask.1`?

Reply:
There are a few nice things, like an updated Babel config: https://github.com/MetaMask/snaps/blob/main/packages/snaps-cli/CHANGELOG.md

But its more of a process question, should the template use the latest Flask release of CLI or the latest non Flask 🤔